src/function/table/table_scan.cpp

```diff
@@ -421,40 +536,20 @@ static unique_ptr<FunctionData> TableScanDeserialize(Deserializer &deserializer,
 		throw SerializationException("Cant find table for %s.%s", schema, table);
 	}
 	auto result = make_uniq<TableScanBindData>(catalog_entry.Cast<DuckTableEntry>());
-	deserializer.ReadProperty(103, "is_index_scan", result->is_index_scan);
+	deserializer.ReadDeletedProperty<bool>(103, "is_index_scan");
```

Comment:
Can we repurpose the `is_index_scan` property by doing `CastNoConst` in the `InitGlobal` on the bind data and then setting it if we are doing an index scan? Then we can use this in `TableScanToString`